Documentation/configuring-storage.md

```diff
@@ -50,6 +50,27 @@ data:
 ```
 The `spec.storage.hive.azure.container` should be the container you wish to store metering data at, and the optional `spec.storage.hive.azure.rootDirectory` should be the folder you want your data in, inside the container.
 
+## Storing data in Google Cloud Storage
+
+You can also store your data in Google Cloud Storage, and to do so, you must use an existing bucket.
+Edit the `spec.storage` section in the example [gcs-storage.yaml][gcs-config] configuration.
+Set the `spec.storage.hive.gcs.bucket` and `spec.storage.hive.gcs.secretName` with the secret following this format:
+```
+apiVersion: v1
+kind: Secret
+metadata:
+  name: your-gcs-secret
+data:
+  gcs-service-account.json: "c2VjcmV0Cg=="
+
+```
+You can easily encode the entire json by calling
+
+```
+echo -n 'JSON_FILE_HERE'| base64 -w 0
```

Comment:
Might be easier to instruct them to use `kubectl create secret generic your-gcs-secret --from-file gcs-service-account.json=/path/to/your/service-account-key.json`